From 48abf2166daf1aa8b40deb629de65d19e4cb0013 Mon Sep 17 00:00:00 2001
From: Nicolas Le Bayon <nicolas.le.bayon@st.com>
Date: Tue, 5 Jul 2022 15:55:27 +0200
Subject: [PATCH 178/954] fix(st-ddr): phy: disable simulation options for
 training

Disable SimulationOnlyReset and SimulationOnlyTraining bits of MsgMisc
field in message block, to ensure correct timing between Reset and CKE.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 78daf59a074c65c13b56bce880c76d4327e84d9c fix(st-ddr): phy: disable simulation options for training]
Signed-off-by: Nicolas Le Bayon <nicolas.le.bayon@st.com>
Change-Id: Ibf0b7b79fb515376bbfad6b0b7125c61e1745d1d
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/259463
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 drivers/st/ddr/phy/phyinit/src/ddrphy_phyinit_initstruct.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/st/ddr/phy/phyinit/src/ddrphy_phyinit_initstruct.c b/drivers/st/ddr/phy/phyinit/src/ddrphy_phyinit_initstruct.c
index 89e7464255..08bae4d205 100644
--- a/drivers/st/ddr/phy/phyinit/src/ddrphy_phyinit_initstruct.c
+++ b/drivers/st/ddr/phy/phyinit/src/ddrphy_phyinit_initstruct.c
@@ -49,7 +49,7 @@ void ddrphy_phyinit_initstruct(void)
 	 * These are typically invariant across pstate
 	 * ##############################################################
 	 */
-	uint8_t msgmisc = 0x06U;		/*For fast simulation */
+	uint8_t msgmisc = 0x00U;	/* For fast simulation */
 	uint8_t reserved00 = 0x0U;	/*
 					 * Set reserved00[7] = 1 (If using T28 attenuated receivers)
 					 * Set reserved00[6:0] = 0 (Reserved; must be set to 0)
-- 
2.53.0

